src/components/TodoList/TodoList.tsx

```diff
@@ -1,21 +1,91 @@
-import React from 'react';
+import cn from 'classnames';
+import React, { useCallback } from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[],
+  currentTodo: Todo | null,
+  setCurrentTodo: (todo: Todo | null) => void,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  currentTodo,
+  setCurrentTodo,
+}) => {
+  const handleCurrentTodo = useCallback((todo: Todo) => {
+    setCurrentTodo(todo);
+  }, [currentTodo]);
```

Comment:
Callback code doesn't depend on currentTodo value.
```suggestion
  const handleCurrentTodo = useCallback((todo: Todo) => {
    setCurrentTodo(todo);
  }, []);
```